jni/utilities/wavereader.cpp

```diff
@@ -48,7 +48,7 @@ AudioBuffer* WaveReader::fileToBuffer( std::string inputFile )
 
         if ( !strcmp( id, "RIFF" ))
         {
-            fread( &size, sizeof( unsigned long ), 1, fp );
+            fread( &size, sizeof( unsigned int ),  1, fp );
             fread( id,    sizeof( unsigned char ), 4, fp );
```

Comment:
_unsigned long_ confirmed to be 8 bytes (64-bit) in length on ARM64, _unsigned int_ should report correctly as 4 bytes (32-bit) in length across all supported architectures.